packages/embed/e2e/spec/functional/chat.spec.ts

```diff
@@ -0,0 +1,27 @@
+describe(`Chat Popover`, () => {
+  before(() => {
+    cy.visit('/chat')
```

Comment:
I see there are some failing tests because this returns 404. I think it should be `/chat.html`.